packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -108,8 +108,28 @@ sealed class _DebugSemanticsRoleChecks {
     SemanticsRole.tab => _semanticsTab,
     SemanticsRole.tabBar => _semanticsTabBar,
     SemanticsRole.tabPanel => _noCheckRequired,
+    // TODO(chunhtai): add checks when the roles are used in framework.
+    // https://github.com/flutter/flutter/issues/159741.
+    SemanticsRole.searchBox => _unimplemented,
```

Comment:
This is the test that if one remove the enum, this will throw compile time error